pom.xml

```diff
@@ -63,6 +63,11 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
+        <groupId>org.codehaus.sonar.sslr-squid-bridge</groupId>
+        <artifactId>sslr-squid-bridge</artifactId>
+        <version>2.7-SNAPSHOT</version>
```

Comment:
Please depend on released version 2.6. This will fix the build on travis and avoid the mess around building the project locally you encountered. (basically maven was not able to download version 2.7-SNAPSHOT from the central repo as we don't publish SNAPSHOT version).